recipes/libxml2/all/conanfile.py

```diff
@@ -355,6 +355,12 @@ def _module_file_rel_path(self):
                             "conan-official-{}-variables.cmake".format(self.name))
 
     def package_info(self):
+        # FIXME: cmake creates LibXml2::xmllint imported target for the xmllint executable
+        self.cpp_info.set_property("cmake_file_name", "LibXml2")
+        self.cpp_info.set_property("cmake_target_name", "LibXml2::LibXml2")
+        self.cpp_info.builddirs.append(self._module_subfolder)
```

Comment:
Actually I've just realized that it was not even needed to add our generated module file to builddirs, because it's just an internal detail of `cmake_find_package` generator.